pages/events.tsx

```diff
@@ -0,0 +1,205 @@
+import { Accordion, Box, DarkMode, Flex, Grid, Input } from "@chakra-ui/react";
+import { DevRelEvent } from "components/devRelEvents/DevRelEvent";
+import { FeaturedCard } from "components/devRelEvents/FeaturedCard";
+import { Aurora } from "components/homepage/Aurora";
+import { HomepageTopNav } from "components/product-pages/common/Topnav";
+import { HomepageSection } from "components/product-pages/homepage/HomepageSection";
+import { NextSeo } from "next-seo";
+import { PageId } from "page-id";
+import { useState } from "react";
+import { Heading } from "tw-components";
+import { ThirdwebNextPage } from "utils/types";
+
+const featuredEvents = [
+  {
+    type: "Hackathon",
+    title: "Ready Player 3 Hackathon",
+    date: "Jan 16th - 31st",
+    banner: "/assets/og-image/readyplayer3.png",
+    link: "https://thirdweb.com/hackathon/readyplayer3",
+  },
+];
+
+const allEvents = [
```

Comment:
some of these are now already in the past => we'll need a toggle between "upcoming" and "past"

Reply:
hmm a dropdown?

Reply:
a select should do it, default value should be "upcoming"